prettier.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  bracketSpacing: false,
+  printWidth: 80,
```

Comment:
printWidth, semi, tabWidth, and useTabs are all set to their defaults here:
https://prettier.io/docs/en/options.html

Should we just omit those?

Reply:
There are here because you added them to `.prettierrc`. I actually prefer sticking to the defaults  as much as possible.

Reply:
+1 to sticking to the defaults.  Single quotes are likely the only one that would be hella disruptive.